File: packages/CVMTests/scripts/node/graphics_test.js
//name: NodeJS Graphics
//description: graphics output column input
//language: nodejs
//input: dataframe df
//input: column xName {type:numerical; allowNulls:false} [Column x]
//input: column yName {type:numerical; allowNulls:false} [Column y]
//output: graphics rect

$$.svg("<svg><rect width=180 height=190 style='fill: blue;'/></svg>");

File: packages/CVMTests/scripts/octave/graphics_test.m
#name: Octave Graphics
#description: graphics output column input
#language: octave
#input: dataframe df
#input: column xName {type:numerical; allowNulls:false}
#input: column yName {type:numerical; allowNulls:false}
#output: graphics barchart

pkg load dataframe;

d = dataframe(df);

bar(d(1:3,[xName]));
